Mathlib/Algebra/Group/Action/Basic.lean

```diff
@@ -146,3 +147,19 @@ variable [Group α] [Monoid β] [MulAction α β] [SMulCommClass α β β] [IsSc
   ⟨fun h => inv_smul_smul g m ▸ h.smul g⁻¹, IsUnit.smul g⟩
 
 end SMul
+
+lemma le_pow_smul {G : Type*} [Monoid G] {α : Type*} [Preorder α] {g : G} {a : α}
```

Comment:
I think this should go somewhere under `Mathlib.Algebra.Order.Group`; maybe in a new file. CC @YaelDillies for suggestions

Reply:
Agreed. `Algebra.Group.something` should be order-free.

Reply:
Good point. I've moved them to `Algebra.Order.Group.Action`. Still had to add an import, but having `Algebra.Order.Group.Action` import `Algebra.Group.Action.Defs` doesn't seem so bad.

Reply:
But let me know if there's a better home.